tests/console/firewalld.pm

```diff
@@ -0,0 +1,140 @@
+# SUSE's openQA tests
+#
+# Copyright © 2019 SUSE LLC
+#
+# Copying and distribution of this file, with or without modification,
+# are permitted in any medium without royalty provided the copyright
+# notice and this notice are preserved.  This file is offered as-is,
+# without any warranty.
+
+# Summary: Test FirewallD basic usage
+# Maintainer: Alexandre Makoto Tanno <atanno@suse.com>
+
+use strict;
+use warnings;
+use base "consoletest";
+use testapi;
+use utils qw(systemctl);
+
+sub run {
+    select_console("root-console");
+
+    # Check Service State, enable it if necessary, set default zone to public
+    record_info 'Check Service State';
+    assert_script_run("if [ `systemctl is-active firewalld | grep -q -i 'inactive' ; echo $?` -eq 0 ]; then systemctl start firewalld; fi");
+    assert_script_run("firewall-cmd --set-default-zone=public");
+
+
+    # Test #1 - Stop firewalld then start it
+    record_info 'Test #1', 'Test: Stop firewalld, then start it';
+    systemctl('stop firewalld');
+    systemctl('start firewalld');
+
+    # Check Service State, enable it if necessary
+    record_info 'Check Service State';
+    assert_script_run("if [ `systemctl is-active firewalld | grep -q -i 'inactive' ; echo $?` -eq 0 ]; then systemctl start firewalld; fi");
```

Comment:
`if systemctl is-active firewalld | grep -q -i 'inactive'; then systemctl start firewalld; fi`
can be shorter and do the same

Reply:
I do agree with you. I'm checking this test's code and changing it according to your suggestion.